### ISSUE TITLE
Mudança de papel por perfil não autorizado

### ISSUE THREAD

**Issue by almeidafab** [servicosgovbr/editor-de-servicos#493]
Logado como Publicador, consegui alterar o perfil de um Administrador para Editor. O mesmo acontece também logado como Ponto Focal, consegui alterar o perfil de um Administrador para Publicador ou Editor.

**Comment by gustavo-freitas** [comment]
@almeidafab isso não acontece mais. Fizemos modificações essa manhã no ambiente de desenvolvimento. Por favor, peço que verifique novamente.

**Comment by almeidafab** [comment]
@gustavo-freitas mesmo com o botão de editar oculto eu consigo alterar o perfil clicando no nome. Nesse caso, entrei como Publicador e consegui alterar o Ponto Focal para Editor, inclusive consegui alterar o nome dele também.

![captura de tela de 2015-12-30 10 35 15](https://cloud.githubusercontent.com/assets/9214263/12050777/20799d24-aee1-11e5-89fa-656165b845f7.png)

**Comment by gustavo-freitas** [comment]
@almeidafab Certo. Iremos desabilitar o acesso pelo nome na lista de usuários.

**Comment by josselmo** [comment]
Funcionou. OK
